plugin/pkg/scheduler/schedulercache/node_info.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package schedulercache
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/api"
+)
+
+// NodeInfo is node level aggregated information.
+// Note: if change fields, also change String() method
+type NodeInfo struct {
+	PodNum int
+	// Total requested resource of all pods (including assumed ones) on this node
+	RequestedResource *Resource
+}
+
+// Resource is a collection of compute resource .
+type Resource struct {
+	MilliCPU int64
+	Memory   int64
+}
+
+type PodInfo struct {
+	RequestedResource Resource
+}
+
```

Comment:
general comment on consistency of commenting functions and structs.  Should apply comments to all.